HDF5Examples/config/cmake/HDFExampleMacros.cmake

```diff
@@ -31,13 +31,13 @@ macro (BASIC_SETTINGS varname)
     set (CMAKE_MFC_FLAG 0)
   endif ()
 
-  set (CMAKE_C_STANDARD 99)
+  set (CMAKE_C_STANDARD 11)
```

Comment:
The examples need to work for all versions of the library - should this be conditional on the lib version?

Reply:
We can leave it at 99. I don't think the current examples do anything special for C11.